hrm-service/service-tests/contacts.test.ts

```diff
@@ -244,10 +237,7 @@ describe('/contacts route', () => {
 
         const expected = expectedGetContact || contact;
 
-        const res = await request
```

Comment:
Up until now we've just been supressing prettier on type imports, so that's not ideal either.

Personally I'd happily ditch prettier and just use eslint, but I don't think that's the prevailing opinion! 😅

I would say supress the issues for this PR to keep it manageable - you could have a prettier bump and associated reformats as a follow up PR if you wanted

Reply:
Annnd of course prettier didn't revert all of the changes it forced after a downgrade. I could find the change-set and try to revert it, I guess?

Reply:
Detangling the rest of the prettier changes would be rough. It isn't *that* many files compared to the upgrade. Basically changes in these few files can be ignored during PR review as they are all prettier related. Is it alright to leave those in place in this PR?
<img width="394" alt="Screen Shot 2022-10-20 at 10 29 23 AM" src="https://user-images.githubusercontent.com/7569921/196977599-7d23dddd-4a51-44b2-aba0-68ce9c6d0419.png">